java/src/main/java/com/google/oak/client/OakClient.java

```diff
@@ -16,13 +16,13 @@
 
 package com.google.oak.client;
 
+import com.google.oak.evidence.Evidence;
 import com.google.oak.util.Result;
-import java.util.Optional;
 
 /**
- * Abstract client for sending and receiving encrypted messages to a server. This class provides an
- * abstract builder that must be implemented by concrete subclasses of this class. The builder takes
- * a {@code RpcClientProvider}, an {@code EncryptorProvider}, and optionally an {@code
+ * Abstract client for sending and receiving encrypted messages to a server. We recommend concrete
```

Comment:
What do we mean by "recommend"? Is there another option? Similar, the builders "should use". Again, is there another option?

Reply:
Thanks. Updated the description.